tutorials/baseline/auto_trainer.py

```diff
@@ -0,0 +1,111 @@
+import os
+from minio import Minio
+from tqdm.auto import tqdm
+
+import joblib
+import pandas as pd
+
+from typing import Union
+
+import numpy as np
+import scipy.sparse as sp
+import torch
+from torch.utils.data import TensorDataset
+
+from sklearn.feature_extraction.text import ENGLISH_STOP_WORDS
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+from knodle.model.logistic_regression_model import LogisticRegressionModel
+from knodle.trainer.auto_trainer import AutoTrainer
+
+# This python script contains rarely any explanation. For more description, we refer to the corresponding
+# jupyter notebook. There the steps are explained in more detail
+
+
+# Define some functions
```

Comment:
some small function descriptions are still lacking to me

Reply:
From above: "This python script contains rarely any explanation. For more description, we refer to the corresponding jupyter notebook. There the steps are explained in more detail."

Reply:
yes, I have seen it, and it is ok that there is no detailed explanation - but I think one-line comments for the next two functions would be good (as it is for create_tfidf_dataset function). But not so important actually :)

Reply:
I'm not sure where. np_to_tensor_dataset should be rather descriptive?